qa/tasks/cephfs/test_openfiletable.py

```diff
@@ -0,0 +1,43 @@
+import time
+from cephfs_test_case import CephFSTestCase
+from teuthology.exceptions import CommandFailedError
+from tasks.cephfs.cephfs_test_case import CephFSTestCase, for_teuthology
+
+class OpenFileTable(CephFSTestCase):
+    CLIENTS_REQUIRED = 1
+    MDSS_REQUIRED = 1
+
+    def test_max_items_per_obj(self):
+        """
+        The maximum number of openfiles omap objects keys are now equal to
+        osd_deep_scrub_large_omap_object_key_threshold option.
+        """
+        self.fs.mon_manager.raw_cluster_cmd('config', 'set', 'mds', 'osd_deep_scrub_large_omap_object_key_threshold', '5')
+
+        # Write some bytes to a file
+        size_mb = 1
+
+        # Hold the file open
+        file_count = 6
+        for i in xrange(0, file_count):
+            filename = "open_file{}".format(i)
+            p = self.mount_a.open_background(filename)
+            self.mount_a.write_n_mb(filename, size_mb)
+
+        time.sleep(5)
+
+        """
+        With osd_deep_scrub_large_omap_object_key_threshold value as 5 and
+        opening 6 files we should have a new rados object with name
+        mds0_openfiles.1 to hold the extra keys.
+        """
+
+        stat_out = self.fs.rados(["stat", "mds0_openfiles.1"])
+
+        # Now close the file
+        self.mount_a.kill_background(p)
+
+        # Unlink the dentry
+        for i in xrange(0, file_count):
+            filename = "open_file{}".format(i)
+            self.mount_a.run_shell(["rm", "-f", filename])
```

Comment:
Between each test the file system is deleted; no need to clean up.

Reply:
Done.